lib/globus_client/identity.rb

```diff
@@ -20,7 +20,8 @@ def get_identity_id(user_id)
     def exists?(user_id)
       get_identity_id(user_id)
       true
-    rescue
+    # if no active user is returned
+    rescue RuntimeError
```

Comment:
Don't rescue an `UnexpectedResponse::UnauthorizedError` anymore since we want to retry upon that exception.